sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestRegisteredMappersWork.java

```diff
@@ -102,6 +103,25 @@ public void setColor(String color)
         }
     }
 
+    void foo() throws Exception {
+        Jdbi jdbi = dbRule.getJdbi();
+        jdbi.useHandle(handle -> {
+            BeanMappingDao dao = handle.attach(BeanMappingDao.class);
+            TestColumnMappers.SomeBeanDao another = handle.attach(TestColumnMappers.SomeBeanDao.class);
+
+
+        });
+
+        BeanMappingDao dao = jdbi.onDemand(BeanMappingDao.class);
+        dao.useHandle(h -> {
+            TestColumnMappers.SomeBeanDao another = h.attach(TestColumnMappers.SomeBeanDao.class);
+            Bean foo = dao.findByName("foo");
+
+            foo.setColor("brown");
+            dao.insertBean(foo);
+        });
+    }
```

Comment:
Whoops, I did not mean to check this in.. was showing a coworker the correct way to negotiate on-demand with shared handles.

Reply:
Will remove this in a follow up PR for other docs.